scripts/simple.py

```diff
@@ -0,0 +1,7 @@
+def uftrace_entry(args):
```

Comment:
It's not needed here but I think it's ok to add.  Maybe it can be used to control python version (2 or 3) if we add python3 support later.